### PR TITLE
Python script to find missing translations

### LINKED ISSUE #818: Missing a lot of translations
In the last releases in uTox UI was added a lot new stuff, and some of new functions are missing translations.

I wrote down a small [script](https://gist.github.com/redmanmale/18748ca46ac15e2ec41cad53257a8c96#file-getmissingtranslations-py) (in Python 3) to get all missing translations and I was surprised to see how many (25 langs and ~2.5k fields).

Also I noticed, that danish lang file is the only one which named unlike the others.

Full list of missing translations is [here](https://gist.github.com/redmanmale/18748ca46ac15e2ec41cad53257a8c96#file-utox_missing_translations_report_for_0-14-0-txt).

### PR DESCRIPTION
**Pull request by redmanmale**
To make it easier to fix #818.

---
This change is [Reviewable](https://reviewable.io/reviews/utox/utox/820)

### CODE REVIEW

File: extra/common/get_missing_translations.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        report.write('---------------------------\nMissing translations for ' + lang + ':\n---------------------------\n')
+        for const in consts:
+            report.write(const + '\n')
+        report.write('\n')
+
+os.chdir(os.path.dirname(__file__))
+reportFile = os.path.join(os.getcwd(), 'utox_missing_translations_report.txt')
+
+langs = []
+etalonLang = 'en'
```
Comment (robinlinden) [inline extra/common/get_missing_translations.py]: Maybe rename this to referenceLang or something to make it clearer. I've no idea what etalon stands for. :P

**Comment by robinlinden** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/820)*

File: extra/common/get_missing_translations.py (hunk 1 above)
Comment (redmanmale) [inline extra/common/get_missing_translations.py]: My bad. I mean 'etalon' is for 'reference', 'standart'.
I'll fix this.

Comment (robinlinden) [inline extra/common/get_missing_translations.py]: No worries. Thank you so much for reporting so many bugs and the PR and everything, btw. :) We really appreciate it.

File: extra/common/get_missing_translations.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        return ''
+
+def getStringConsts(lang):
+    print('Getting string consts for ' + lang, end='\n')
+    langFile = readFileFromUrl('https://raw.githubusercontent.com/uTox/uTox/develop/langs/'+ lang + '.h')
+    constsFinder = re.compile('msgid\((\w+)\)')
+    consts = constsFinder.findall(langFile)
+    return consts
+
+def getMissingConsts(etalonConsts, translateArr):
```
Comment (robinlinden) [inline extra/common/get_missing_translations.py]: Missed the etalon in this function, line 45, and line 51 though. :P

**Comment by robinlinden** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/820#-:-KhYg7f44bJnb9WqM6HI:bnfp4nl)*

**Comment by GrayHatter** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/820#-:-KhYnqYEHgAbcrzeBMbM:bnfp4nl)*

**Comment by endoffile78** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/820#-:-Khd06Z9ft4jKekpxHWy:bnfp4nl)*

**Pull request merged**